examples/va/hci/values-ci-framework.yaml

```diff
@@ -0,0 +1,205 @@
+# REMOVE_ME
+# Provided as a better alternative to this:
+#   https://github.com/fultonj/architecture/tree/ci-framework
+# But should be genereated by ci-framework in the future
+
+# Replace all `enp6s0` below with output of this command:
+#   oc debug node/master-0 -- chroot /host nmcli \
+#      -g GENERAL.DEVICE,IP4.ADDRESS device show | \
+#      grep -B 1 168.122 | head -1
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: network-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  # nodes
+  node_0:
+    name: master-0
+    internalapi_ip: 172.17.0.5
+    tenant_ip: 172.19.0.5
+    ctlplane_ip: 192.168.122.10
+    storage_ip: 172.18.0.5
+  node_1:
+    name: master-1
+    internalapi_ip: 172.17.0.6
+    tenant_ip: 172.19.0.6
+    ctlplane_ip: 192.168.122.11
+    storage_ip: 172.18.0.6
+  node_2:
+    name: master-2
+    internalapi_ip: 172.17.0.7
+    tenant_ip: 172.19.0.7
+    ctlplane_ip: 192.168.122.12
+    storage_ip: 172.18.0.7
+
+  # networks
+  ctlplane:
+    dnsDomain: ctlplane.example.com
+    subnets:
+      - allocationRanges:
+          - end: 192.168.122.120
+            start: 192.168.122.100
+          - end: 192.168.122.200
+            start: 192.168.122.150
+        cidr: 192.168.122.0/24
+        gateway: 192.168.122.1
+        name: subnet1
+    prefix-length: 24
+    iface: enp6s0
+    mtu: 1500
```

Comment:
I had to change this to `9000` in order for the NNCPs to successfully configure the interfaces for `storage` and `storageMgmt`.  Otherwise I saw this error in the NMState logs on the OCP nodes:

```
NmstateError: VerificationError: Verification failure: storage.interface.mtu desire '9000', current '1500'
```

I think that's because the base interface has to have MTU >= the MTU you desire to set on any VLAN interface attached to it.

Reply:
Good catch. I've updated values.yaml so the interface has 9000.

Also, I changed the mtu values in lib from _replaced_ to 1500 so that they are substituted as integers, not strings.